Mathlib/Data/Pi/Algebra.lean

```diff
@@ -16,6 +17,10 @@ import Mathlib.Data.Sum.Basic
 This file provides basic definitions and notation instances for Pi types.
 
 Instances of more sophisticated classes are defined in `Pi.lean` files elsewhere.
```

Comment:
This file now also includes non-notation instances

Reply:
The doc still looks accurate to me?

Reply:
Now in #10693